http/obmc_shell.hpp

```diff
@@ -179,6 +179,15 @@ inline void requestRoutes(App& app)
         .onopen([](crow::websocket::Connection& conn,
                    const std::shared_ptr<bmcweb::AsyncResp>&) {
             BMCWEB_LOG_DEBUG << "Connection " << &conn << " opened";
+
+            // TODO: this user check must be removed when WebSocket privileges
+            // work.
+            if (conn.getUserName() != "service")
+            {
+                BMCWEB_LOG_DEBUG
+                    << "only service user have access to obmc_shell";
+                conn.close("only service user have access to bmc console");
```

Comment:
If the console session is established by the 'service' user, why there is a need to check or allow 'service' user to close?
I'm not sure whether it is possible for one user to setup console and another user close, so it is just for clarification.

Reply:
the service user has access to obmc_shell, which means if we get any other uses, we will restrict that user.
WebSocket privilege is core dumping that why this is a hack for restricting users.